gputop-csv/gputop-csv.js

```diff
@@ -456,19 +515,42 @@ function write_rows(metric, accumulator)
     }
 }
 
+var flag = 0;
```

Comment:
Maybe you find a better name for this.
Using a boolean type seems appropriate too.
Probably make it part of the GputopCSV instance.

Reply:
Thanks, I have renamed this parameter with "warning_once" and put it into GputopCSV instance.